upf/upf_cli.c

```diff
@@ -253,15 +253,12 @@ upf_nat_pool_add_del_command_fn (vlib_main_t * vm,
   u8 *nwi_name;
   u8 *nwi_s = 0;
   ip4_address_t start, end;
-  u16 min_port;
-  u16 max_port;
-  u16 port_block_size;
+  u32 min_port = UPF_NAT_MIN_PORT;
```

Comment:
Changing to u32 to fit unformat functions looks good
But let's also change generic usage of min/max ports here.